### PR TITLE
doc: parameter tags[]

### PR DESCRIPTION
**Pull request by kadhikari**
Ticket: https://jira.kisio.org/browse/NAVP-946

### CODE REVIEW

**Review by nonifier — APPROVED** [review]

**Review by xlqian — APPROVED** [review]

File: documentation/slate/source/includes/apis.md [hunk 1]
```diff
@@ -1936,8 +1937,12 @@ For example:
 
 -   overall public transport traffic report on Ile de France coverage
     -   <https://api.navitia.io/v1/coverage/fr-idf/traffic_reports>
+-   overall public transport traffic report on Ile de France coverage with disruptions having tags passed in parameter values
+    -   <https://api.navitia.io/v1/coverage/fr-idf/traffic_reports?tags[]=bob&tags[]=bobette>
```
Comment (mbounabi) [inline documentation/slate/source/includes/apis.md]: Bob and Bobette in the official doc??? Seems more professional to give actual possible tags.

File: documentation/slate/source/includes/apis.md [hunk 2]
```diff
@@ -1936,8 +1937,12 @@ For example:
...
 -   Is there any perturbations on the RER network ?
     -   <https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/traffic_reports>
+-   Is there any perturbations on the RER network with disruptions having tags passed in parameter values ?
+    -   <https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/traffic_reports?tags[]=bob&tags[]=bobette>
```
Comment (mbounabi) [inline documentation/slate/source/includes/apis.md]: Same comment for Bob and Bobette. Even though they're nice and welcome in our tests, it doesn't fell like they're in the right place here.

**Review by mbounabi — APPROVED** [review]

**Comment by mbounabi** [comment]
Wait for the feature to be merged before merging the doc

**Review by kinnou02 — APPROVED** [review]

File: documentation/slate/source/includes/apis.md [hunk 3]
```diff
@@ -1936,8 +1937,12 @@ For example:
 
 -   overall public transport traffic report on Ile de France coverage
     -   <https://api.navitia.io/v1/coverage/fr-idf/traffic_reports>
+-   overall public transport traffic report on Ile de France coverage with disruptions having tags passed in parameter values
+    -   <https://api.navitia.io/v1/coverage/fr-idf/traffic_reports?tags[]=incidents&tags[]=alerte>
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: I would put 'alert' in :gb: (unless it mentions a search giving results) : "alerts" (no final `e`)
And 'incident' singular (no final `s` ) to be consistent

File: documentation/slate/source/includes/apis.md [hunk 4]
```diff
@@ -1936,8 +1937,12 @@ For example:
...
 -   Is there any perturbations on the RER network ?
     -   <https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/traffic_reports>
+-   Is there any perturbations on the RER network with disruptions having tags passed in parameter values ?
+    -   <https://api.navitia.io/v1/coverage/fr-idf/networks/network:RER/traffic_reports?tags[]=incidents&tags[]=alerte>
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md]: same here

**Review by benoit-bst — APPROVED** [review]

**Pull request merged**